cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js

```diff
@@ -5,28 +5,28 @@ import { Given, Then } from "cypress-cucumber-preprocessor/steps";
 Then('the button to download At a Glance should appear at the bottom of the page', () => {
     cy.get(`.ap-button-block`).should('be.visible');
     cy.get(`.ap-button-block a[target='_blank']`).should('be.visible');
-    });
+});
 
-    And('a PDF icon located at {string} should be displayed with a text {string}', (iconUrl,iconText) => {
-    
-        const baseURL = Cypress.config('baseUrl');
-        cy.document().then((document) => {
-            const element = document.querySelector("div[class='ap-button-block'] a");
-            const style = window.getComputedStyle(element, '::before');
-            const image = style.getPropertyValue('background-image');
-            const expectedURL= `url("${baseURL}${iconUrl}")`;
-            expect( expectedURL).equal(image);
-           const text=element.innerText;
-           expect(text).equal(iconText);
-        });
+And('a PDF icon located at {string} should be displayed with a text {string}', (iconUrl, iconText) => {
+    const baseURL = Cypress.config('baseUrl');
+    cy.document().then((document) => {
+        const element = document.querySelector("div[class='ap-file-block pdf'] a");
+        const style = window.getComputedStyle(element, '::before');
+        const image = style.getPropertyValue('background-image');
+        const expectedURL = `url("${baseURL}${iconUrl}")`;
+        const newImage = image.replace(/\?v=\w+/, ''); // here is the change
```

Comment:
please remove the comments ))) they were meant to explain the ticket and should not be reflected in the code

Reply:
Done